protocols/relay/src/priv_client/handler.rs

```diff
@@ -489,10 +489,7 @@ impl ConnectionHandler for Handler {
         }
 
         // Update keep-alive handling.
-        if matches!(self.reservation, Reservation::None)
-            && self.alive_lend_out_substreams.is_empty()
-            && self.circuit_deny_futs.is_empty()
-        {
+        if matches!(self.reservation, Reservation::None) && self.circuit_deny_futs.is_empty() {
```

Comment:
`circuit_deny_futs` also contains streams so it is implicitly covered. You can remove this bit!

Reply:
removed b87b155